### ISSUE TITLE
[Accessibility] The 'Clear all (x)' button in the combo box is not accessible by keyboard.

### ISSUE THREAD

**Issue by DandelionWenjing** [ng-select/ng-select#2042]
**Describe the bug**
The 'Clear all (x)' button in the combo box is not accessible by keyboard.

**To Reproduce** 
Steps to reproduce the behavior (if example is not provided):
1. Go to ng select component.
2. Select options and the clear all button on the right display.
3. Tried to use "Tab" button to focus the clear all while can not focus by keyboard.

**Expected behavior**
When the clear all button display, can use the keyboard to focus and do the operation.

**Screenshots**
![image](https://user-images.githubusercontent.com/33156672/153690870-4e1b7b14-7066-40c7-b351-d1d084a90a6d.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Angular Version 12.2.13
 - Ng Select Version 7.4.0

**Smartphone (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Angular Version 12.2.13
 - Ng Select Version 7.4.0

**Additional context**
